tests/phpunit/payment-methods/class-wc-stripe-upe-payment-gateway-test.php

```
@@ -3990,4 +3990,227 @@ public function provide_test_is_valid_optimized_checkout_page() {
+				'amount_total' => 2000,
+			]
+		);
+		WC_Stripe_Database_Cache::set( 'checkout_session_' . $checkout_session_id, $checkout_session );
+
+		$formatted_total = '$10.00';
+		$result          = $this->mock_gateway->add_converted_currency_information( $formatted_total, $order );
+
+		WC_Stripe_Database_Cache::delete( 'checkout_session_' . $checkout_session_id );
+
+		$this->assertEquals( $formatted_total, $result );
+	}
```

PHPMD (2.15.0)

[error] The method test_add_converted_currency_information_returns_unchanged_total_when_no_presentment_details is not named in camelCase. (undefined)

(CamelCaseMethodName)